packages/atomic/src/components/search/atomic-generated-answer/atomic-generated-answer.pcss

```diff
@@ -31,14 +60,24 @@
 }
 
 .source-citations {
+  --height: 2.2em;
+  --index-height: 20px;
+  --max-citation-width: 170px;
+
+  .citations-label {
+    /* Making sure the label height is the same as the citation for vertical centering. */
+    height: var(--height);
+  }
+
   .citations-container {
     .citation {
-      max-width: 160px;
+      max-width: var(--max-citation-width);
```

Comment:
Are you sure about limiting the width of citations this way? In the video, 4 of the 5 citations are labelled `How to Resolve...` making it impossible for a user to know what differs between all of them. There is still some free space on the right. The citations could be a little wider and maybe allow the user to see the difference between each of them.

Reply:
Yeah I'm not super confident about this being the greatest solution though I think we definitely need to truncate them as most titles are way too long. In my latest commit I made the citation element a css part so an implementer could set their own max-width. Do you have a better idea?
FYI i also increased it from 160 to 170 so what you see in the video is 160.

Reply:
I agree with you, most titles are long and should be truncated. Would it be possible to adjust the citation width automatically to take all available width, with a maximum of 4 citations on a single row?

For instance:
- 1 citation: would take 100% width
- 2 citations: would take 50% each
- 4 citations: would take 25% each
- 5 citations: the first 4 would take 25% each, and the last one 100% on a second row (we may want to make them all the same size tough to keep them aligned)

What do you think?

Reply:
I like the general idea! To account for mobile or IPX style interfaces we would probably also need a min-width as 25% of an iphone width is probably just a few letters.

Reply:
<img width="894" alt="Screenshot 2023-07-18 at 3 28 19 PM" src="https://github.com/coveo/ui-kit/assets/16785453/ae4c5738-7bb2-4a64-b5fe-1de6bb5be753">

So close! It would be ideal if the last one could max out at the content width instead of the container width. Might be reaching the limits of flexbox though

Reply:
Can't get that width to cap at available space OR content-width :/ I spoke with UX and the citations in general will be looked at for improvements in the very near future so I'll hold off on modifying them for now.

Reply:
No worries. Thanks for looking into it!